src/harness/reference_models/iap/iap.py

```diff
@@ -221,17 +247,17 @@ def performIapForEsc(protected_entity, sas_uut_fad_object, sas_th_fad_objects):
 
   # Get azimuth radiation pattern
   ant_pattern = esc_point['azimuthRadiationPattern']
-
-  # Initialize array of size length of ant_pattern list
-  ant_pattern_gain = np.arange(len(ant_pattern))
-  for i, pattern in enumerate(ant_pattern):
-    ant_pattern_gain[i] = pattern['gain']
+  ant_pattern = sorted([(pat['angle'], pat['gain']) for pat in ant_pattern])
+  angles, gains = zip(*ant_pattern)
+  if angles != tuple(range(360)):
+      raise ValueError('ESC pattern inconsistent')
+  ant_gain_pattern = np.array(gains)
 
   # Get ESC antenna information
   esc_antenna_info = interf.EscInformation(antenna_height=esc_point['height'],
                        antenna_azimuth=esc_point['antennaAzimuth'],
-                       antenna_gain=esc_point['antennaGain'],
-                       antenna_pattern_gain=ant_pattern_gain)
+                       antenna_gain=0,
```

Comment:
remove the field `antenna_gain`. 
I think i removed it from the EscInformation in the latest code on the master (submitted today) as part of another PR.

Reply:
Removed   antenna_gain